pandas/core/arrays/categorical.py

```diff
@@ -2099,6 +2099,10 @@ def _reverse_indexer(self) -> dict[Hashable, npt.NDArray[np.intp]]:
     # ------------------------------------------------------------------
     # Reductions
 
+    def _reduce_and_wrap(self, name: str, *, skipna: bool = True, kwargs):
+        result = self._reduce(name, skipna=skipna, **kwargs)
+        return type(self)([result], dtype=self.dtype)
```

Comment:
do we get here with e.g. any/all?

Reply:
Categorical doesn't support any/all, IDK why actually, seems like it could, if the categories do. 

Do you have any specific issue in mind?

Reply:
gentle ping...

Reply:
maybe a comment that if any/all are ever supported then we shouldnt do this wrapping?